src/license/license.proto

```diff
@@ -65,6 +70,8 @@ message ClusterStatus {
 message UpdateClusterRequest {
   string id = 1;
   string address = 2;
+  string user_address = 3;
```

Comment:
FYI @actgardner , I just added the fields to update cluster as well

Reply:
Good catch!